pyproject.toml

```diff
@@ -49,14 +49,14 @@ python = "^3.7"
 
 poetry-core = "^1.3.2"
 poetry-plugin-export = "^1.1.2"
-"backports.cached-property" = { version = "^1.0.2", python = "<3.8" }
+"backports.cached-property" = { version = "^1.0.2", python = "3.7" }
```

Comment:
I like <3.8 even if we don't support 3.6. I suppose that's a matter of taste. 🤷

Reply:
The intention is to make it grepable -- I agree that `<3.8` is generally preferable, but in this specific case of covering one version I think it makes it easier to remove later.